validator/project_validator.go

```diff
@@ -217,7 +226,7 @@ func CheckProjectWarnings(project *model.Project, yamlBytes []byte) ValidationEr
 }
 
 // verify that the project configuration syntax is valid
-func CheckProjectErrors(project *model.Project, includeLong bool) ValidationErrors {
+func CheckProjectErrors(project *model.Project, projectConfig *model.ProjectConfig, includeLong bool) ValidationErrors {
```

Comment:
I did this since I had considered there might be places where we call CheckProjectErrors outside of the logic of the validation route where we'd want to also validate the ProjectConfig.  On second thought though I think it's best to just keep this in the validation route (similar to how we only check for project warnings in the validation route) and leave CheckProjectErrors unchanged.